emscripten-snapshot-compiler/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# Build emscripten snapshot compiler
+#
+
+PYTHON ?= python3
+JERRYSCRIPT_ROOT ?= $(abspath $(CURDIR)/../jerryscript)
+BUILD_DIR := $(CURDIR)/build
+
+# emscripten WebAssembly toolchain
+# https://emscripten.org/index.html
+ifdef EMSDK
+export EMSCRIPTEN ?= $(EMSDK)/upstream/emscripten
+endif
+
+ifndef EMSCRIPTEN
+	$(error 'EMSCRIPTEN' undefined - please configure emsdk)
```

Comment:
@mikee47 May be you should remove the tab in line 16, similar to line 12, otherwise the makefile execution ends with the following error:
```
Makefile:16: *** recipe commences before first target.  Stop.
```